cmd/hamctl/command/rollback_test.go

```diff
@@ -0,0 +1,181 @@
+package command_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/lunarway/release-manager/cmd/hamctl/command"
+	"github.com/lunarway/release-manager/cmd/hamctl/command/actions"
+	"github.com/lunarway/release-manager/internal/artifact"
+	"github.com/lunarway/release-manager/internal/git"
+	internalhttp "github.com/lunarway/release-manager/internal/http"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRollback(t *testing.T) {
+	var (
+		serviceName = "some-service-name"
+		env         = "dev"
+	)
+	var describeReleaseResponse func() []internalhttp.DescribeReleaseResponseRelease
+
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var releaseRequest internalhttp.ReleaseRequest
+		_ = json.NewDecoder(r.Body).Decode(&releaseRequest)
+
+		switch {
+
+		case strings.Contains(r.URL.Path, "describe/release/"):
+			resp := internalhttp.DescribeReleaseResponse{
+				Service:     serviceName,
+				Environment: env,
+				Releases:    describeReleaseResponse(),
+			}
+			err := json.NewEncoder(w).Encode(resp)
+			require.NoError(t, err)
+		case strings.Contains(r.URL.Path, "release") && r.Method == http.MethodPost && releaseRequest.ArtifactID == "artifact-1":
+			resp := internalhttp.ReleaseResponse{
+				Service:       serviceName,
+				ReleaseID:     "some-release-id",
+				Status:        "released",
+				ToEnvironment: env,
+				Tag:           "dev",
+			}
+
+			err := json.NewEncoder(w).Encode(resp)
+			require.NoError(t, err)
+
+		case strings.Contains(r.URL.Path, "release") && r.Method == http.MethodPost && releaseRequest.ArtifactID == "artifact-0":
+			resp := internalhttp.ReleaseResponse{
+				Service:       serviceName,
+				ReleaseID:     "some-release-id",
+				Status:        "released",
+				ToEnvironment: env,
+				Tag:           "dev",
+			}
+
+			err := json.NewEncoder(w).Encode(resp)
+			require.NoError(t, err)
+
+		default:
+			require.Fail(t, "http url was not found")
+			w.WriteHeader(http.StatusNotFound)
+		}
+	}))
+
+	c := internalhttp.Client{
+		BaseURL: server.URL,
+	}
+
+	gitConfigAPI := git.GitConfigAPIMock{
+		CommitterDetailsFunc: func() (string, string, error) {
+			return "name", "email", nil
+		},
+	}
+
+	releaseClient := actions.NewReleaseHttpClient(&gitConfigAPI, &c)
+
+	runCommand := func(selectRollback command.SelectRollbackRelease, t *testing.T, args ...string) ([]string, error) {
```

Comment:
Test functions usually have `t` as their first argument to make it super clear they only work inside tests. WDYT?

```suggestion
	runCommand := func(t *testing.T, selectRollback command.SelectRollbackRelease, args ...string) ([]string, error) {
```